pinc/project_states.inc

```diff
@@ -22,17 +22,171 @@ include_once($relPath.'site_vars.php');
 
   (Actually, the naming convention is in transition.)
 */
+global $PROJECT_STATES_IN_ORDER, $project_state_medium_label_, $project_state_long_label_,
+$project_state_phase_, $project_states_for_star_metal_;
 
-$PROJECT_STATES_IN_ORDER = [];
-$project_state_medium_label_ = [];
-$project_state_long_label_ = [];
-$project_state_forum_ = [];
-$project_state_phase_ = [];
-$project_states_for_star_metal_ = [
-    'BRONZE' => [],
-    'SILVER' => [],
-    'GOLD' => [],
-];
+function define_project_states($rounds, $pools)
+{
+    global $waiting_projects_forum_idx, $pp_projects_forum_idx, $posted_projects_forum_idx,
+    $projects_forum_idx, $completed_projects_forum_idx, $deleted_projects_forum_idx,
+    $project_states_for_star_metal_, $Round_for_project_state_;
+
+    $PROJECT_STATES_IN_ORDER = [];
+    $project_state_medium_label_ = [];
+    $project_state_long_label_ = [];
+    $project_state_forum_ = [];
+    $project_state_phase_ = [];
+    $project_states_for_star_metal_ = [
+        'BRONZE' => [],
+        'SILVER' => [],
+        'GOLD' => [],
+    ];
+
+    $project_round_states = [
+        ["BAD_PROJECT", "bad", "bad", _("Bad Project"), _("Bad Project")],
```

Comment:
It would be possible to write the translated strings like `_("%s: Bad Project")` where %s will be replaced by something like "P1" in the short case or _('Proofreading Round 3').